Jack/Property.fs

```diff
@@ -229,6 +247,9 @@ module Property =
         else
             failure
 
+    let counterexample (msg : string) : Property<unit> =
+        Gen.constant (Journal.singleton msg, Success ()) |> ofGen
```

Comment:
I'm not 100% on this, but I changed `counterexample` to simply be a `Property<unit>` rather than a function `Property<'a> -> Property<'a>` because it makes calling it inside a computation expression possible. You would think this doesn't matter because we have the custom operation `counterexample`, but for some reason custom operations can't be used at the same time as control flow operations like `for` and `while`.